its/ruling/src/test/resources/expected/python-S5915.json

```diff
@@ -3,19 +3,4 @@
     641,
     1482,
   ],
-  'project:tensorflow/python/data/experimental/kernel_tests/take_while_test.py':[
```

Comment:
We probably wouldn't mind raising on these, even if we can't be sure it is `unittest`. Even in another framework, the pattern is probably a mistake.

Reply:
I removed the call to `UnittestUtils.isWithinUnittestTestCase` in order to have those usecase raised again.
Indeed check on the name itself seems quite enough to highlight that this is very probably a mistake, the risk of FP seems unlikely to me. Not sure if we should do the same elsewhere in the end, but this is maybe defeating the purpose of the `isWithinUnittestTestCase` itself ^^'

Not sure how the tensorflow project is working : it seems to have it's own unittest framework implementation with specific name, but methods and usage are the same.